src/app/examples/links/links.html

```diff
@@ -1,14 +1,26 @@
 <a href="#">Default Link</a><br />
 <a href="#" target="_blank">Link in new Window</a><br />
 
-<a href="#"><i class="link-icon link-start element-right-2 flip-rtl"></i>Default Icon Link</a><br />
+<a href="#"
+  ><i aria-hidden="true" class="link-icon link-start element-right-2 flip-rtl"></i>Default Icon
+  Link</a
```

Comment:
```suggestion
<a href="#">
 <i aria-hidden="true" class="link-icon link-start element-right-2 flip-rtl"></i>
  Default Icon Link
</a
```

Would be nice if you can change the formatting everywhere to this. Unless this causes to many white spaces